activity/rest/activity.go

```diff
@@ -119,7 +120,21 @@ func (a *RESTActivity) Eval(context activity.Context) (done bool, err error) {
 		req.Header.Set("Content-Type", contentType)
 	}
 
-	client := &http.Client{}
+	// Set the proxy server to use, if supplied
+	proxy := context.GetInput(ivProxy)
+	var client *http.Client
+	if proxy != nil && proxy.(string) != "" {
+		proxyURL, urlErr := url.Parse(proxy.(string))
+		if urlErr != nil {
+			log.Debug("Error parsing proxy url, skipping proxy:", urlErr)
+			client = &http.Client{}
```

Comment:
Are we allowing to continue with at bad proxy? can't this bring undesired behavior?